backend/tests/unit/agents/test_kubernetes_agent.py

```
@@ -407,6 +409,21 @@ async def test_execute_mcp_tools_server_validation(self, kubernetes_agent_with_m
         agent, mock_llm, mock_mcp, mock_registry = kubernetes_agent_with_mocks
         agent._configured_servers = ["kubernetes-server"]
         
+        # Mock call_tool to perform validation
+        async def mock_call_tool_with_validation(
+            server_name, tool_name, parameters, session_id=None,
```

Ruff (0.14.3)

Unused function argument: `parameters`

(ARG001)